packages/code-du-travail-ui/src/LargeLink.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export default function LargeLink({ children, ...props }) {
+  return (
+    <a className="btn-large" {...props}>
+      {children}
+    </a>
+  );
+}
```

Comment:
(marche aussi)

```js
export default function LargeLink(props) {
  return (
    <a className="btn-large" {...props}/>
  );
}
```

Reply:
yup but I need to add `eslint-disable-next-line  jsx-a11y/anchor-has-content`